### ISSUE TITLE
PCA loadings or equivalent

### ISSUE THREAD

**Issue by TonyKess** [Rosemeis/HaploNet#4]
Is it possible to get per SNP or per haplotype loadings (or something similar) for selection inference when using the PCA method?

**Comment by Rosemeis** [comment]
Yes, you can get the haplotype cluster loadings out using "--loadings" in the *haplonet pca* command. :-)

**Comment by TonyKess** [comment]
Thanks! Can I assume chi-square distribution for eventually calculating pvalue/FDR on these?
